language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/SortingUtil.java

```diff
@@ -85,7 +88,7 @@ public static boolean isModuleCompletionItem(LSCompletionItem item) {
      * @return {@link Boolean} whether type completion or not
      */
     public static boolean isTypeCompletionItem(LSCompletionItem item) {
-        return (item instanceof SymbolCompletionItem
+        return (item.getType() == LSCompletionItem.CompletionItemType.SYMBOL
```

Comment:
`json`, `type`, `handle` are some of the Symbol  CompletionItems  that do not have an associated symbol. Those symbols are not considered as typeCompletionItems in this logic.